src/livetvtuner.html

```diff
@@ -63,7 +63,6 @@ <h1 class="sectionTitle">${HeaderLiveTvTunerSetup}</h1>
                 </div>
 
                 <p class="drmMessage hide">${DrmChannelsNotImported}</p>
-                <p class="otherOptionsMessage hide" style="margin:2em 0;">${ForAdditionalLiveTvOptions}</p>
```

Comment:
Apologies, I hadn't noticed this usage. The Live TV page may throw an error and crash if this element is missing so we need to see if that controller still looks for this and remove the check.